integration-libs/cdc/schematics/add-cdc/index_spec.ts

```diff
@@ -122,6 +122,44 @@ describe('Spartacus CDC schematics: ng-add', () => {
   });
 
   describe('CDC feature', () => {
+    describe('warning for missing jsSDKUrl', () => {
+      let firstMessage: string | undefined;
+      beforeEach(async () => {
+        schematicRunner.logger.subscribe((log) => {
+          if (!firstMessage) {
+            firstMessage = log.message;
+          }
+        });
+
+        appTree = await schematicRunner
+          .runSchematicAsync('ng-add', { ...cdcFeatureOptions }, appTree)
+          .toPromise();
+      });
+
+      it('show the warning', () => {
```

Comment:
to semantically follow the naming of assertion tests, let us put it as `should show the warning`

Reply:
agreed